### DISCUSSION TITLE
search: sorting (remory / default)

### DISCUSSION THREAD

**Discussion by KamiyaMinoru** [InfinityLoop1308/PipePipe#228] [Ideas]
1. 不知道有沒有可能針對經常搜尋的關鍵字也可以變成訂閱項目
2. 因為搜尋的時候常常要找的是最新影片, 但目前的搜尋功能有點複雜
　要先搜尋關鍵字然後再點放大鏡再選擇選項再點放大鏡
　希望能夠有記住偏好或者直接讓搜尋功能有選單不用特地點開之類的UI

**Comment by InfinityLoop1308** [comment]
> 不知道有沒有可能針對經常搜尋的關鍵字也可以變成訂閱項目

Not possible, but recent search should have existed as history for quick searching.

> 讓搜尋功能有選單不用特地點開之類的UI

I don't think such UI exists considering the layout

> 希望能夠有記住偏好

You may want to open a issue for setting the default search filter, but it wouldn't be high priority as this is a small user case.
